packages/react-forms-dom/src/components/text.tsx

```diff
@@ -55,9 +55,9 @@ export class Text extends BaseDomField<TextProps, BaseDomFieldState, HTMLInputEl
         }
     }
 
-    protected get RawDefaultValue(): string {
-        if (this.props.defaultValue != null) {
-            return this.props.defaultValue;
+    protected GetRawDefaultValue(props: TextProps): string {
+        if (props.defaultValue != null) {
+            return props.defaultValue;
         }
 
```

Comment:
I'd rather keep it. And for inconsistencies, I think we'll have to create a proper TSLint rule for this sometime later.